.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Global rule:
+* @rhuairahrighairidh @karzak @pirtleshell @drklee3 @nddeluca @DracoLi @evgeniy-scherbina @sesheffield
```

Comment:
🏆 excellent add.
can you include a comment referencing [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) for easy access/explanation of this for those not in the know?